demos/fleet_provisioning/fleet_provisioning_csr/CMakeLists.txt

```
@@ -0,0 +1,50 @@
+set( DEMO_NAME "fleet_provisioning_csr_demo" )
```

(NIt) Using the folder name at least if not the target name containing the `with` prefix makes it easier to read the purpose of the demo
```suggestion
set( DEMO_NAME "fleet_provisioning_with_csr_demo" )
```